app/scripts/views/complete_sign_up.js

```diff
@@ -65,6 +69,7 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       return self.fxaClient.verifyCode(uid, code)
           .then(function () {
             self.logScreenEvent('verification.success');
+            self._verificationExperiment.performAction('verification');
```

Comment:
Does `performAction` mean `do this action now` or `this action has been performed`. If the latter, the name `performAction` could be made more clear.